README.md

```diff
@@ -1465,6 +1467,7 @@ Update Time, five active automations, webhooks.
   * [Penpot](https://penpot.app) - Web-based, open-source design and prototyping tool. Supports SVG. Completely free.
   * [pexels.com](https://www.pexels.com/) - Free stock photos for commercial use. Has a free API that allows you to search photos by keywords.
   * [photopea.com](https://www.photopea.com) — A Free, Advanced online design editor with Adobe Photoshop UI supporting PSD, XCF & Sketch formats (Adobe Photoshop, Gimp and Sketch App).
+  * [photostockeditor.com](https://photostockeditor.com) — Search +350k Free Stock Pictures for comercial use.
```

Comment:
Same story here, I am not 100% up to speed with these rules - its been a few years since I was in photography commercially - but for me these sites are quite sketchy.